### ISSUE TITLE
Consider editing or removing the note for 'set of web pages'

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag2ict#418]
### Currently

In the [definition of 'set of web pages'](https://www.w3.org/TR/wcag2ict-22/#applying-set-of-web-pages-to-non-web-documents-and-software), the words "simply substitute" are confusing. It sounds like an instruction for the reader, but WCAG2ICT already does the substitution.

<blockquote>

Note

For success criteria that use the term “set of web pages”, either explicitly or implicitly ([2.4.1](https://www.w3.org/TR/wcag2ict-22/#bypass-blocks), [2.4.5](https://www.w3.org/TR/wcag2ict-22/#multiple-ways), [3.2.3](https://www.w3.org/TR/wcag2ict-22/#consistent-navigation), and [3.2.4](https://www.w3.org/TR/wcag2ict-22/#consistent-identification)), simply substitute "set of non-web documents" and "set of software programs" when applying this to non-web technologies.
</blockquote>

### Proposal 1

<blockquote>

Note

For success criteria that use the term “set of web pages”, either explicitly or implicitly ([2.4.1](https://www.w3.org/TR/wcag2ict-22/#bypass-blocks), [2.4.5](https://www.w3.org/TR/wcag2ict-22/#multiple-ways), [3.2.3](https://www.w3.org/TR/wcag2ict-22/#consistent-navigation), and [3.2.4](https://www.w3.org/TR/wcag2ict-22/#consistent-identification)), "set of web pages" is replaced by "set of non-web documents" and "set of software programs" when applying this to non-web technologies.
</blockquote>

### Proposal 2

Remove the note.

### Proposal 3

Simplify the note , more like the note for 'Web page', like this:

> For those success criteria that use the term "set of Web pages", WCAG2ICT provides specific replacement terms for "set of Web pages".

**Comment by mitchellevan** [comment]
PR #417 adds missing 3.2.6 to this note, but that PR can be abandoned if we decide to remove the note.

**Comment by bruce-usab** [comment]
I agree the current phrasing warrants improvement.  I like 1 or 3 much better than 2 (removing the note).

**Comment by maryjom** [comment]
The task force made some edits to Proposal 1 to simplify and remove the list of SCs using the term. These lists are already included in the key terms "set of software programs" and "set of documents". We reached a [resolution on the changes](https://www.w3.org/2024/07/11-wcag2ict-minutes#r03) and incorporated proposal 1c as shown in the minutes in PR #441.

**Comment by GreggVan** [comment]
Can you paste what the resolution is here?  I  tried following all the links and never could find what  1c actually said.
thanks.

**Comment by maryjom** [comment]
@GreggVan If you scroll up about 6 lines from the resolution link, you can read the text we reached resolution on - or by looking at PR 441. Phil just reviewed the PR this morning and I'm merging today after my back-to-back meetings this morning.

**Comment by maryjom** [comment]
Closing this issue as resolved by PR #441.
